tox.ini

```diff
@@ -198,6 +198,7 @@ commands_pre =
   python -m pip install -e {toxinidir}/opentelemetry-api[test]
   python -m pip install -e {toxinidir}/opentelemetry-semantic-conventions[test]
   python -m pip install -e {toxinidir}/opentelemetry-sdk[test]
+  python -m pip install protobuf==3.19.4
```

Comment:
Worth adding the link to the issue here?

Reply:
Will this need to be pinned as well? https://github.com/open-telemetry/opentelemetry-python/blob/e280a8aa8524b1a74270d4be553360bba07a29c8/opentelemetry-proto/setup.cfg#L45

Reply:
For some reason, the issue only affects linting so it the change probably has something to do with typing annotations. Also I pinned only `tox` because this only affects our builds and not any actual behavior of our instrumentations/components that depend on `protobuf`.

Reply:
Also added the link to the issue.